src/main/java/com/_up/megastore/services/implementations/ProductService.java

```diff
@@ -57,20 +57,36 @@ private String saveProductImage(MultipartFile multipartFile) {
     private Product getVariantOf(UUID variantOfId) {
         return variantOfId != null ? findProductByIdOrThrowException(variantOfId) : null;
     }
+    @Override
+
+
+
     public ProductResponse updateProduct(UUID productId, UpdateProductRequest updateProductRequest, MultipartFile multipartFile){
```

Comment:
¿Por qué tantas líneas vacías? Podrías dejarlo así

```suggestion
    @Override
    public ProductResponse updateProduct(UUID productId, UpdateProductRequest updateProductRequest, MultipartFile multipartFile) {
```
Es importante que el código quede legible